### ISSUE TITLE
build_all_ios.sh:  x86_64 compilation failed.

### ISSUE THREAD

**Issue by xingliejin** [tensorflow/tensorflow#15942]
when i compile build_all_ios.sh:   

****tensorflow 1.1 :**   is OK.... build success....**

when i load "xxxx.pb" model,  error:

Invalid argument: No OpKernel was registered to support Op 'Mul' with these attrs.  Registered devices: [CPU], Registered kernels:
  device='CPU'; T in [DT_FLOAT]
	 [[Node: UpSample2D_2/mul = Mul[T=DT_INT32](UpSample2D_2/mul/x, UpSample2D_2/Const)]]

How to solver ?

**tensorflow 1.4 :**   

make: *** [/Users/open/Downloads/tensorflow-1.4/tensorflow/contrib/makefile/gen/host_obj/tensorflow/core/lib/random/distribution_sampler.o] Error 1
+ '[' 2 -ne 0 ']'
+ echo 'i386 compilation failed.'
i386 compilation failed.
+ exit 1
## 
**tensorflow 1.5 :**   

ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [/Users/open/Downloads/tensorflow-1.5/tensorflow/contrib/makefile/gen/bin/ios_X86_64/benchmark] Error 1
+ '[' 2 -ne 0 ']'
+ echo 'x86_64 compilation failed.'
x86_64 compilation failed.
+ exit 1

How to solver ?

**python:3.6.3  mac OS:  10.12.6**

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by xingliejin** [comment]
Have I written custom code : No
OS Platform and Distribution:  python:3.6.3 mac OS: 10.12.6
TensorFlow installed from:  use "pip install tensorflow"  
TensorFlow version: 1.4.0
Bazel version: no Bazel
CUDA/cuDNN version: no cuda ,i'm in mac
GPU model and memory: no

**Comment by xingliejin** [comment]
**How to solver the first question ?**
  when i load "xxxx.pb" model, error:
Invalid argument: No OpKernel was registered to support Op 'Mul' with these attrs. Registered devices: [CPU], Registered kernels:
device='CPU'; T in [DT_FLOAT]
[[Node: UpSample2D_2/mul = Mul[T=DT_INT32](UpSample2D_2/mul/x, UpSample2D_2/Const)]]

**if the tf code simple. the "xxxx.pb" can  load....
when i use  "tf.nn.conv2d_transpose"  , i get the error ....**

**Comment by rlehman221** [comment]
I am having the same error when running tensorflow/contrib/makefile/build_all_ios.sh on my Macbook Pro: 10.13.2
python:3.6.3
TensorFlow version: 1.4.0

**Comment by xingliejin** [comment]
you need delete  compile_ios_tensorflow.sh  " exit 1 "....

**Comment by rlehman221** [comment]
I am really new to all of this, where can I delete that?

**Comment by tensorflowbutler** [comment]
The original poster has replied to this issue after the stat:awaiting response label was applied.

**Comment by drpngx** [comment]
So, we're trying to use tflite for mobile needs. I think it's probably the better supported option. Would that work for you?

**Comment by tensorflowbutler** [comment]
Nagging Assignee @aselle: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @aselle: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @aselle: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @aselle: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by aselle** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
